TaskEngine/Program.cs

```diff
@@ -83,7 +84,7 @@ public static void Main()
             // Start consuming from all queues.
             serviceProvider.GetService<DownloadPlaylistInfoTask>().Consume();
             serviceProvider.GetService<DownloadMediaTask>().Consume();
-            serviceProvider.GetService<ConvertVideoToWavTask>().Consume();
+            // serviceProvider.GetService<ConvertVideoToWavTask>().Consume();
```

Comment:
It is too easy to accdidentally  undelete the "//" ; delete this line or turn it into a comment that cant be compiled.